### PR TITLE
chore(docs): Updated tutorial deployment step.

### PR DESCRIPTION
**Pull request by jsumnersmith**
👋 Hey all, with the launch of Hosting, we wanted to let users deploy on Gatsby Cloud instead of Surge during Step One of our tutorial. Edits and suggestions welcome.

### CODE REVIEW

File: docs/docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -425,7 +381,11 @@ When you created a new Gatsby project with a starter, it automatically made an i
 git push -u origin master
 ```
 
-Now you're ready to link this GitHub repository right to Gatsby Cloud! Check out the reference guide on [Deploying to Gatsby Cloud](/docs/how-to/previews-deploys-hosting/deploying-to-gatsby-cloud/#set-up-an-existing-gatsby-site).
+Now you're ready to link this GitHub repository right to Gatsby Cloud! Sign-in to Gatsby Cloud using the Github login option. If this is your first time logging into Gatsby Cloud, you will create a workspace where you can add multiple sites.
```
Comment (DSchau) [inline docs/docs/tutorial/part-one/index.md:384]:
```suggestion
Now you're ready to link this GitHub repository right to Gatsby Cloud! Sign-in to Gatsby Cloud using the login option (GitHub or Gitlab). If this is your first time logging into Gatsby Cloud, you will create a workspace where you can add multiple sites.
```

**Review by DSchau — APPROVED** [review]
I share your broader point that _maybe_ this is deploying too quickly (e.g. wrong place to have a deploy step) but that's tangential to this discussion.

File: docs/docs/tutorial/part-one/index.md [hunk 2]
```diff
@@ -363,53 +363,9 @@ The Gatsby `<Link />` component is for linking between pages within your site. F
...
-
-Open the web address listed on the bottom line (`lowly-pain.surge.sh` in this
-case) and you'll see your newly published site! Great work!
-
-### Alternative: Deploying to Gatsby Cloud
-
-[Gatsby Cloud](https://gatsbyjs.com) is a platform built specifically for Gatsby sites, with features like real-time previews, fast builds, and integrations with dozens of other tools. It's the best place to build and deploy sites built with Gatsby, and you can use Gatsby Cloud free for personal projects.
-
-To deploy your site to Gatsby Cloud, create an account on [GitHub](https://github.com) if you don't have one. GitHub allows you to host and collaborate on code projects using Git for version control.
+To deploy your site to Gatsby Cloud, first create an account on [GitHub](https://github.com) (or [Gitlab](https://gitlab.com)) if you don't have one. These platforms allow you to host and collaborate on code projects using Git for version control.
```
Comment (meganesu) [inline docs/docs/tutorial/part-one/index.md:368]:
```suggestion
To deploy your site to Gatsby Cloud, first create an account on [GitHub](https://github.com) (or [GitLab](https://gitlab.com)) if you don't have one. These platforms allow you to host and collaborate on code projects using Git for version control.
```

File: docs/docs/tutorial/part-one/index.md [hunk 3]
```diff
@@ -425,7 +381,11 @@ When you created a new Gatsby project with a starter, it automatically made an i
 git push -u origin master
 ```
 
-Now you're ready to link this GitHub repository right to Gatsby Cloud! Check out the reference guide on [Deploying to Gatsby Cloud](/docs/how-to/previews-deploys-hosting/deploying-to-gatsby-cloud/#set-up-an-existing-gatsby-site).
+Now you're ready to link this GitHub repository right to Gatsby Cloud! Sign-in to Gatsby Cloud using the login option (GitHub or Gitlab). If this is your first time logging into Gatsby Cloud, you will create a workspace where you can add multiple sites.
```
Comment (meganesu) [inline docs/docs/tutorial/part-one/index.md:384]:
```suggestion
Now you're ready to link this GitHub repository right to Gatsby Cloud! Sign-in to Gatsby Cloud using the login option (GitHub or GitLab). If this is your first time logging into Gatsby Cloud, you will create a workspace where you can add multiple sites.
```

**Review by meganesu — DISMISSED** [review]
Thanks for updating this! Left a few small #branding comments.

For transparency, I'm planning on updating the rest of the tutorial to be v3. It'll be very similar to the structure of the Getting Started workshop from GatsbyConf.

I think it's fine for them to deploy to Gatsby Cloud in the first section. That way, they can get all the setup out of the way at the beginning, and they'll be able to see the value add of new commits triggering new builds.

**Review by meganesu — APPROVED** [review]

**Comment by jsumnersmith** [comment]
@meganesu Not sure what the merge process is for docs. Ok to just merge to master? Or do I need to coordinate with the core team?

**Comment by DSchau** [comment]
Let's merge.

**Pull request merged**